### ISSUE TITLE
Could you release the training, validation and testing dataset splits for Adobe-DPE

### ISSUE THREAD

**Issue by dallon-an** [sjmoran/DeepLPF#5]
Thank you for your amazing work.
But I'm sorry that the link in the readme is broken:
https://www.cmlab.csie.ntu.edu.tw/project/Deep-Photo-Enhancer/%5BOnline_Demo_Models%5D_Deep-Photo-Enhancer.zip

So could you please release the training, validation and testing dataset splits for Adobe-DPE

**Comment by sjmoran** [comment]
Hi, thank you for your interest in our work. Unfortunately we do not have access to the original zip and dataset splits (we have searched over the past week to see if we could help here). For the supervised training and evaluation, it could be that the first 2250 images are the training dataset (in numeric order as you see on the Adobe5K website, left column: https://data.csail.mit.edu/graphics/fivek/), with the last 500 as the test set i.e. images 4500-5000 are the test dataset. However, best confirm this fact by reaching out to the DPE authors. We used the identical splits from their publication.
